packages/ember-model/lib/model.js

```diff
@@ -398,6 +398,32 @@ Ember.Model = Ember.Object.extend(Ember.Evented, {
 Ember.Model.reopenClass({
   primaryKey: 'id',
 
+  create: function(properties) {
+    var transformedProperties = {};
+    if (properties) {
+      var attributes = this.getAttributes();
+
+      for (var i=0; i<attributes.length; i++) {
+        var attribute = attributes[i];
+        var transformedKey;
+        var meta = this.metaForProperty(attribute);
+
+        if (meta.options && meta.options.key) {
+          transformedKey = this.camelizeKeys ? underscore(meta.options.key) : meta.options.key;
+        } else {
+          transformedKey = this.camelizeKeys ? underscore(attribute) : attribute;
+        }
+
+        transformedProperties[transformedKey] = properties[attribute];
+      }
+    }
+
+    var obj = this._super.apply(this, arguments);
```

Comment:
Is this any different to calling this._super directly?

Reply:
You mean like `this._super(...arguments)`? `ember-model` doesn't use ES6 and I wanted to avoid beginning to use it